concent_api/concent_api/settings/base.py

```diff
@@ -229,6 +229,12 @@
 # A global constant defining the path to self-signed SSL certificate to storage cluster
 STORAGE_CLUSTER_SSL_CERTIFICATE_PATH = ''
 
+# A global constant defining amount of time needed for geth to create new chain segment
+CREATION_NEW_CHAIN_SEGMENT_TIME = 15  # seconds
```

Comment:
Please add system_check which validates if those values are set when `PAYMENT_BACKEND` is set to `sci_backend`. Please also validate type (int and url).

Reply:
and unit tests for this check

Reply:
Both done